src/Psalm/ErrorBaseline.php

```diff
@@ -112,15 +112,19 @@ public static function read(FileProvider $fileProvider, string $baselineFile): a
 
                 $issueType = $issue->tagName;
 
-                $files[$fileName][$issueType] = [
-                    'o' => (int)$issue->getAttribute('occurrences'),
-                    's' => [],
-                ];
+                $files[$fileName][$issueType] = ['o' => 0, 's' => []];
                 $codeSamples = $issue->getElementsByTagName('code');
 
                 foreach ($codeSamples as $codeSample) {
+                    $files[$fileName][$issueType]['o'] += 1;
                     $files[$fileName][$issueType]['s'][] = trim($codeSample->textContent);
                 }
+
+                // TODO: Remove in v6.0.0
```

Comment:
```suggestion
                // TODO: Remove in Psalm 6
```

This is easier to grep for.